Makefile

```diff
@@ -0,0 +1,7 @@
+install:
```

Comment:
Do you know help command in Makefile ?


Add this line in top of the file
```
help: ## Display available commands
	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Add comment on action like
```
install: ## Setup all project dependencies

start: ## Start container in dev environment
```

test this command:
```
make
make help
```